### ISSUE TITLE
Update AssemblyFileVersion only

### ISSUE THREAD

**Issue by onovotny** [GitTools/GitVersion#913]
Is there a way to only update the `AssemblyFileVersion` with whatever goes there? I'm running into situations where we're going to have to manually version the AssemblyVersion and it cannot even be the same within the package. I do still want to stamp the descriptive file version though.

The real scenario is in a NuGet package with multiple `netstandard` implementations. Suppose you provide a `netstandard1.0` and a `netstandard1.3` version. The later version includes more surface area. In this case, you can't really use the same `AssemblyVersion` because of binding issues. There's really no good answer here, but one approach is to keep the assembly version in "bands" --

Like the Netstandard 1.0 version will be 1.0.0-1000, and the netstandard 1.3 version will be 1.0.1001-2000, etc.
Basically you always have to ensure the one with more surface area is a higher number. You can't even really increment the major/minor per release because in the above scenario, a 1.1.0 (netstandard 1.0) would be considered higher than the 1.0.1500 (netstandard 1.3) version and be "expected" to have all of its api area, which it can't.

Thoughts?

**Comment by onovotny** [comment]
See https://github.com/Reactive-Extensions/Rx.NET/issues/205 for a full explanation of the issue.

**Comment by JakeGinnivan** [comment]
Will get this released asap
